docs/production-setup.md

```diff
@@ -217,34 +288,60 @@ In order to be able to propose batches we are going to register our Ethereum acc
 to do this execute this command:
 
 ```bash
-docker run --rm -v /$HOME/zkevm-node/config.toml:/app/config.toml hermeznetwork/zkevm-node:latest sh -c "./zkevm-node register --cfg=/app/config.toml --network=internaltestnet --y <public IP or URL for users to access the sequencer> "
+docker run --rm -v /$HOME/zkevm-node/config.toml:/app/config.toml \
+    -v /$HOME/zkevm-node/genesis.json:/app/genesis.json \
+    hermeznetwork/zkevm-node:latest \
+    sh -c "./zkevm-node register --cfg=/app/config.toml --y <public IP or URL for users to access the sequencer> "
 ```
```

Comment:
This was related to the 1.5, where the sequencer must register before sending sequences, this is not required for the current version.

Reply:
+++